src/Umbraco.Web.UI.Login/src/components/layouts/new-password-layout.element.ts

```diff
@@ -0,0 +1,176 @@
+import type { UUIButtonState, UUIInputPasswordElement } from '@umbraco-ui/uui';
+import { CSSResultGroup, LitElement, css, html, nothing } from 'lit';
+import { customElement, property, query, state } from 'lit/decorators.js';
+import { umbAuthContext } from '../../context/auth.context';
+
+@customElement('umb-new-password-layout')
```

Comment:
A "page" is used in the router to render a "layout". The idea was to separate the two types of components from each other. I'm not sure it makes sense to introduce form components as well since they are only really used in one layout.